cookie-banner-rules-list.json

```diff
@@ -1136,11 +1065,11 @@
     },
     {
       "click": {
-        "optIn": "button#privacy-cor-wall-accept",
-        "presence": "div.privacy-cor-wall"
+        "optIn": "button#privacy-cp-wall-accept",
+        "presence": "div.privacy-cp-wall"
       },
       "cookies": {},
-      "id": "caf3084b-dfc5-40ca-b66e-a477913efc67",
+      "id": "b90700d6-32a6-44e2-adf4-46acd54089c9",
```

Comment:
Is this ID change deliberate? Why is it needed?

Reply:
+1

Reply:
It does not need to change, that is a personal error. Only the CSS selectors needed to be changed.